scripts/requirements.txt

```diff
@@ -1,5 +1,5 @@
-breathe>=4.9.1
-sphinx>=1.7.5
-docutils>=0.14
+breathe==4.14.1
+sphinx==2.4.4
+docutils==0.16
```

Comment:
These seem to be "bleeding edge" versions. Except for PR #225 I could build everything just fine with my stock sphinx 2.1.2 

Versions aside, @deb-intel can we please revert to `>=` to force fewer users to have one virtualenv per sphinx project?

Cc: @mmaka1 , @xiulipan, @lgirdwood .

Reply:
@deb-intel , @intelkevinputnam you forgot to update the documentation:
https://thesofproject.github.io/latest/contribute/process/docbuild.html#install-documentation-tools

> Our documentation processing has been tested to run with:
> 
> Python 3.6.3
> Doxygen version 1.8.13
> Sphinx version 1.7.5
> Breathe version 4.9.1
> docutils version 0.14
> sphinx_rtd_theme version 0.4.0
> 

( A more general discussion about versioning happened in #198, thanks again @intelkevinputnam  )